python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -35,8 +40,7 @@ def translate_ccdnum(self, md):
         if md.exists("CCDNUM"):
             ccdnum = md.get("CCDNUM")
         else:
-            self.log.warn("Unable to find value for CCDNUM")
-            ccdnum = None
+            return self._translateFromCalibId("ccdnum", md)
```

Comment:
I am guessing that here -- and in the equivalent date and filter blocks below -- that the `CCDNUM` key is for CP provided calibs and the `CALIB_ID` handling is for LSST? It would be helpful to add comments to that effect.